### ISSUE TITLE
Unable to get serious answers to legitimate petitions

### ISSUE THREAD

**Issue by jmerkin** [WhiteHouse/petitions#6]
It seems that many petitions on serious issues are either ignored or given a weak response that doesn't address the questions raised in the petition.

**Comment by yefim** [comment]
This is neither relevant to the code base nor funny.

**Comment by max-mapper** [comment]
In addition this is also a duplicate of #2 and should be closed

**Comment by Bastlynn** [comment]
Seconded, this is a dupe of #2 and should be closed.

**Comment by Bastlynn** [comment]
No. It fails all three rhetorical persuasive approaches.

**Comment by afomi** [comment]
No, I don't. There are more appropriate and effective means.  Plus, this is our inbox.  You're spamming us, an open source community for these civic applications.

**Comment by Bastlynn** [comment]
Are you simply oblivious to the concept of open source software? It's not the white house's code anymore. It belongs to everyone and they can't take it back now.

This code can now be used by _any_ organization that needs a way to handle shared decision making. You want to do some good regarding this work? Take the code, take the improvements made to it, then open a website to organize Occupy efforts with it and use it to support your opposition efforts. You can enjoy your delicious irony, and we can get some honest work done.

**Comment by Bastlynn** [comment]
Then you are demonstrating your ignorance in the non-governmental organization realm, and in the Drupal realm. There are other websites out there, sure, but you can't exactly go to change.org and ask for a copy of their software to run your own tools for your own topics and needs. It does no harm to have choices available designed to go on existing sites, or to build a website around. The origin of the software as a WH project is irrelevant (see http://en.wikipedia.org/wiki/Genetic_fallacy ), the internet came out of DARPA and yet I see you online. Good things can be created out of questionable roots.
